### PR TITLE
Fix #2695: Answer buttons to the top of the screen

### LINKED ISSUE #2695: Answer buttons to the top of the screen
Originally reported on Google Code with ID 1793
```
I'd really like to have a setting to move the answer buttons (show, again, good etc.)
to the top of the screen. This would make it much easier to use a smart phone to review
the cards with one hand. Now I have to support the phone with my little finger and
bend my thumb into a bad position. With the buttons on the top of the screen I would
be able to grip the phone and use my thumb naturally.

Actually quite many android apps are designed like this, like ASTRO file manager, contacts,
dropbox, etc.
```

Reported by `lassund` on 2013-07-03 14:16:15

### PR DESCRIPTION
**Pull request by madhead**

### CODE REVIEW

File: AnkiDroid/src/main/res/xml/preferences_reviewing.xml [hunk 1]
```diff
@@ -106,6 +106,12 @@
             android:title="@string/button_size"
             app:interval="10"
             app:min="100" />
+        <ListPreference
+            android:defaultValue="0"
```
Comment (hssm) [inline AnkiDroid/src/main/res/xml/preferences_reviewing.xml]: This is invalid. Should be "bottom".

**Comment by hssm** [comment]
The repositioning doesn't work nicely with typing fields:

screenshot_2016-07-25-10-22-39

screenshot_2016-07-25-10-22-43

The button bar changes position as the typing field is hidden on the answer side. I think you will also need to reposition the typing field to appear below the buttons when positioned on top.

**Comment by madhead** [comment]
@hssm, I've updated PR for both flaws.

**Comment by hssm** [comment]
Great, and thanks a lot for all your recent contributions!

**Comment by madhead** [comment]
Thanks for the great app! And I hope to contribute more.

**Pull request merged**